config/locales/en.yml

```diff
@@ -40,3 +40,6 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    change_email:
+      subject: "[%{app_name}] Email changed"
```

Comment:
Is there a way to add comments to these to make it easier for translators, I wonder? I'll have a google later. It may be worth making this "Email address changed" rather than "Email changed" to make it clearer.